requirements-dev.txt

```diff
@@ -5,4 +5,4 @@ cryptography<3.1; python_version < '3.6'
 -r requirements.txt
 pylint==2.6.0; python_version >= '3.6'
 -r https://github.com/ansible/ansible/raw/devel/test/lib/ansible_test/_data/requirements/sanity.validate-modules.txt
-molecule[docker]
+molecule[docker]>=3.2.4
```

Comment:
Seems to not like this.

Reply:
"python 2.7", yeah

Reply:
I don't think that version supports Python 2.7. Seems like it was last supported here https://pypi.org/project/molecule/3.0.2.1/. So gotta keep it the way it was or drop Py 2.7